config/desktop-development.json

```diff
@@ -95,6 +95,8 @@
 		"oauth": true,
 		"perfmon": false,
 		"persist-redux": true,
+		"plans/jetpack-backup": false,
```

Comment:
You don't need to specify this if it's `false` for a particular env. IF you omit this line, it'll be considered that it's `false` anyway.

Reply:
I've discussed this in particular with @enejb. Even though flags default to `false`, we took the approach of being verbose here so that all the flags already in place.